README.md

```diff
@@ -8,14 +8,67 @@ This action allows caching dependencies and build outputs to improve workflow ex
 
 See ["Caching dependencies to speed up workflows"](https://help.github.com/github/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows).
 
+## What's New in V2
```

Comment:
This can simply be `What's new` (without the v2)

Similar to the artifact actions, checkout, and setup-python. Checkout started this pattern, you can see it [here](https://github.com/actions/checkout#whats-new)

You can also add a link to the README of the `v1` for the previous version.